engine/consensus/matching/engine.go

```diff
@@ -49,7 +50,7 @@ type Engine struct {
 	seals                   mempool.IncorporatedResultSeals // holds the seals that were produced by the matching engine
 	missing                 map[flow.Identifier]uint        // track how often a block was missing
 	assigner                module.ChunkAssigner            // chunk assignment object
-	checkingSealing         *atomic.Bool                    // used to rate limit the checksealing call
+	isCheckingSealing       *atomic.Bool                    // used to rate limit the checksealing call
```

Comment:
as you have included the dedicated the method `checkingSealing`, would suggest to update to doc 
```suggestion
	isCheckingSealing       *atomic.Bool                    // used to rate limit the checkingSealing calls
```